src/Pages/GlobalConfigurations/Authorization/Shared/components/K8sObjectPermissions/K8sListItemCard.tsx

```diff
@@ -535,6 +567,8 @@ const K8sListItemCard = ({
                             isLoading={isResourceListLoading}
                             menuPlacement="auto"
                             menuPosition="fixed"
+                            // @ts-expect-error fix this with custom typing or custom select
```

Comment:
If we are ever going to create a custom select then we ought to keep a data attribute with type unknown for passing data to internal select components